### DISCUSSION TITLE
Submit error; Status: 'gpu'; team name: onedayday88

### DISCUSSION THREAD

**Discussion by oneday88** [PaddlePaddle/PaddleSpatial#107] [BaiduKDDCup2022] [answered]
Thanks a lot~ Which had pass the local testing.
By the way, . it will be grateful if the organizers can increase the submit time limits as it seems there a lots of issues in the testing environment

**Comment by oneday88** [comment, marked as answer]
Solving this issues by adding  setting['gpu'] = 0 in the "prepare.py"

**Reply by jlo404** [reply]
After executing **eval("test1.zip")** we get **"Cannot use GPU because there is no GPU detected on your machine."**
![FalloGPU](https://user-images.githubusercontent.com/81811961/171953008-df79397e-8a20-4a40-bb06-bb961f4a169a.png)
In prepare.py "gpu": 0  
Should we install paddlepaddle pkg instead of paddlepaddle-gpu==2.3.0?

We're trying to set up baseline model in paddlepaddle framework.

Thanks in advance
Regards

**Reply by ramber1836** [reply]
Please follow this manual (https://www.paddlepaddle.org.cn/documentation/docs/en/install/index_en.html) to set up PaddlePaddle on your local machine.

**Reply by jlo404** [reply]
Thank you for your response

I'd just want to clarify that I'm following the installation of all the packages for the evaluation following these instructions https://github.com/PaddlePaddle/PaddleSpatial/blob/main/apps/wpf_baseline_gru/kddcup22-sdwpf-evaluation/preinstalled-pkgs/paddlepaddle_env_installed_packages.md 

I will find a way to combine the configuration you indicate with the installation of the ~70 packages.

One doubt, with that package installation, apart from being able to train/test/evaluate the baseline model by default, is the graphical tool PaddlePaddle also installed and ready to be used? Or would it need to be installed in a different environment with other modules incompatible with the 70 mentioned above?

Thank you very much!

**Reply by ramber1836** [reply]
> Thank you for your response
> 
> I'd just want to clarify that I'm following the installation of all the packages for the evaluation following these instructions https://github.com/PaddlePaddle/PaddleSpatial/blob/main/apps/wpf_baseline_gru/kddcup22-sdwpf-evaluation/preinstalled-pkgs/paddlepaddle_env_installed_packages.md
> 
> I will find a way to combine the configuration you indicate with the installation of the ~70 packages.
> 

Basically, your local environment has not to be exactly the same as the environment of the online test. 
The list of installed packages is just for your convenience such that you are aware of the PaddlePaddle environment of the online test. You don't need to install all the packages mentioned in the table.  

> One doubt, with that package installation, apart from being able to train/test/evaluate the baseline model by default, is the graphical tool PaddlePaddle also installed and ready to be used? Or would it need to be installed in a different environment with other modules incompatible with the 70 mentioned above?
> 

The PGL is not necessary for the GRU baseline. If you want to try the baseline provided by the official PGL team (https://github.com/PaddlePaddle/PGL/tree/main/examples/kddcup2022/wpf_baseline), then the pgl package is needed and it has been added to the paddlepaddle environment of the online test. To make the pgl compatible with other packages installed on your local machine, you can setup another environment and install PGL.
